viv-wallet-app/src/views/Payment.vue

```diff
@@ -77,25 +75,39 @@ export default defineComponent({
         };
     },
     computed: {
-        amount: function (): number {
+        amount(): number {
             return this.coeff * this.viv;
         },
-        hasBalanceToPay: function (): boolean {
+        hasBalanceToPay(): boolean {
             return this.viv > 0;
         },
     },
+    async mounted() {
+        try {
+            [this.user, this.balance, this.unpaidActions] = await Promise.all([
+                this.usersService.getUser(this.id),
+                this.walletService.getUserBalance(this.id),
+                this.walletService.getUserPayableActions(this.id),
+            ]);
+            this.viv += this.balance;
```

Comment:
it's weird that we do that on the front end anywat... just a comment

Reply:
Yes, I know, but this part will change soon. We're working on it with Benoit.